common/reviews/api/rush-lib.api.md

```diff
@@ -582,6 +582,7 @@ export interface _INpmOptionsJson extends IPackageManagerOptionsJsonBase {
 export interface IOperationExecutionResult {
     readonly cobuildRunnerId: string | undefined;
     readonly error: Error | undefined;
+    readonly executedOnThisAgent: boolean;
```

Comment:
I don't mind having this field, but it does render the status `FromCache` technically redundant.